doc/installation_instructions/basic.rst

```diff
@@ -0,0 +1,300 @@
+     - x86_64
+     - Not supported
+     - 3.7 - 3.9
+   * - Linux
+     - x86_64
+     - Nvidia
+     - 3.7 - 3.9
+
+
+.. note::
+
+    Windows is not supported and there are currently no plans
+    to support Windows.
+
+
+Native support for various machine learning libraries and their
+versions is dictated by our dependency on RedisAI_ 1.2.7. Users
+can also select RedisAI 1.2.3 or 1.2.5 (though that also limits
+the version of the ML libraries).
+
++------------------+----------+-------------+---------------+
+| RedisAI          | PyTorch  | Tensorflow  | ONNX Runtime  |
++==================+==========+=============+===============+
+| 1.2.7 (default)  | 1.11.0   | 2.8.0       | 1.11.1        |
++------------------+----------+-------------+---------------+
+| 1.2.5            | 1.9.0    | 2.6.0       | 1.9.0         |
++------------------+----------+-------------+---------------+
+| 1.2.3            | 1.7.0    | 2.5.2       | 1.9.0         |
++------------------+----------+-------------+---------------+
+
+TensorFlow_ 2.0 and Keras_ are supported through graph freezing_.
+
+ScikitLearn_ and Spark_ models are supported by SmartSim as well
+through the use of the ONNX_ runtime (which is not built by
+default due to issues with glibc on a variety of Linux
+platforms and lack of support for Mac OS X).
+
+.. _Spark: https://spark.apache.org/mllib/
+.. _Keras: https://keras.io
+.. _ScikitLearn: https://github.com/scikit-learn/scikit-learn
+.. _TensorFlow: https://github.com/tensorflow/tensorflow
+.. _PyTorch: https://github.com/pytorch/pytorch
+.. _ONNX: https://github.com/microsoft/onnxruntime
+.. _RedisAI: https://github.com/RedisAI/RedisAI
+.. _freezing: https://github.com/leimao/Frozen-Graph-TensorFlow
+
+------------------------------------------------------------
+
+MacOS-only
+==========
+
+We recommend users and contributors install brew_ for managing installed
+packages.  For contributors, the following brew packages can be helpful
+
+- openmpi_ for building and running parallel SmartRedis examples
+- doxygen_ for building the documention
+- cmake_ for building SmartSim and SmartRedis from source
+
+.. _brew: https://brew.sh/
+.. _openmpi: https://formulae.brew.sh/formula/open-mpi#default
+.. _doxygen: https://formulae.brew.sh/formula/doxygen#default
+.. _cmake: https://formulae.brew.sh/formula/cmake#default
+
+For Mac OS users, the version of ``make`` that comes with the Mac command line
+tools is often 3.81 which needs to be updated to install SmartSim. Users can
+``brew install make`` to get ``make`` > 4.0 but brew will install it as
+``gmake``. An easy way around this is to do ``alias make=gmake``.
+
+.. _from-pypi:
+
+========
+SmartSim
+========
+
+There are two stages for the installation of SmartSim.
+
+ 1. `pip` install SmartSim Python package
+ 2. Build SmartSim using the `smart` command line tool
+
+Step 1: Install Python Package
+==============================
+
+Activate a new virtual environment:
+
+.. code-block:: bash
```

Comment:
Should we include here how to create a new env?

Reply:
Also there is a git clone step for smartredis but not for smartsim, is this correct?

Reply:
conda create -n myenv2 python=3.9 && conda activate myenv2

Reply:
Trying to move away from `conda` for licensing reasons. Here I'm suggesting the using just `pyenv`